libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/accounts/Trips.kt

```diff
@@ -0,0 +1,124 @@
+package com.mapbox.services.android.navigation.v5.internal.accounts
+
+import android.content.SharedPreferences
+import android.os.SystemClock
+import androidx.annotation.NonNull
+import com.mapbox.android.accounts.v1.MapboxAccounts
+
+const val MAPBOX_NAV_PREFERENCE_TRIPS_SKU = "com.mapbox.navigationsdk.accounts.trips.sku"
+const val MAPBOX_NAV_PREFERENCE_ROUTE_REQ_COUNT = "com.mapbox.navigationsdk.accounts.trips.count"
+const val MAPBOX_NAV_PREFERENCE_TRIPS_TIMESTAMP = "com.mapbox.navigationsdk.accounts.trips.time"
+const val DEFAULT_TRIP_REQUEST_COUNT = 0
+const val DEFAULT_TRIP_TOKEN_TIMER = 0L
+
+internal class Trips(
```

Comment:
Have we considered calling it `TripsSku`?

Reply:
Done